opi_video_publishing_policy.md

```diff
@@ -0,0 +1,29 @@
+# Open Programmable Infrastructure (OPI) Video and Presentation approval Process
+
+This policy covers the review of technical content for public presentations and Videos about the Open Programmable Infrastructure (OPI) project. The intent is to have a consistent, accurate, and uniform representation of OPI as well as to assist member companies and individuals.
+
+- Scope
+  - This policy will cover videos, presentations, or documents with OPI technical content that for use in public forums such as industry conferences, academic conferences, or webinars.
+  - Company internal presentations or company customer facing presentations that include OPI information or are about OPI are not covered by this policy
+  - To facilitate reuse a set of common slides and content will be approved jointly with the OPI Outreach committee. This will streamline the process of creating and delivering presentions about OPI as reuse of already approved content will not require a new review.
+  - OPI TSC, Subproject, and Working Group meetin minutes, videos, and recordings do not need approval to post throught https://lists.opiproject.org/ or https://github.com/opiproject.
+<br><br>
+- Process
+  - Upload the content to the OPI google docs presentation area
+  - Request for review by the TSC will be via the TSC mailing list or in person during a TSC meeting
+  - Approval of new content will be voted on by the TSC at the TSC meeting following the request for review
+  - Approved presentations will be posted to opiproject github
+  - Approved videos may be posted to the OPI YouTube Channel
+<br><br>
+- YouTube channel 
+  - management and maintenance will be by the outreach committee
```

Comment:
Add `with oversight by the TSC`.

Reply:
will be helpful to have a link to youtube channel